### PR TITLE
Add Convex integration manifest

### PR DESCRIPTION
**Pull request by atrakh**
**Requirements**

- [x] I have followed the repository's [pull request submission guidelines](../blob/main/README.md#submitting-pull-requests)
- [ ] I have updated my user documentation
- [ ] I have updated my developer documentation (my integration README)
- [x] I have submitted the [Become a Partner](https://launchdarkly.com/partners/) form

This PR is a draft of an integration manifest for an integration between [Convex][https://convex.dev] and LaunchDarkly.

The Convex database will act as a synced feature and segment store for LaunchDarkly flags, allowing users to evaluate flags in the Convex runtime

### CODE REVIEW

File: integrations/convex/manifest.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          "name": "Webhook URL",
+          "description": "The url to the webhook URL exposed by your Convex application.",
+          "type": "string",
+          "placeholder": "e.g. https://wonderful-capybara-515.convex.site/api/launchdarkly-webhook",
+          "isSecret": false
+        },
+        {
+          "key": "signingKey",
+          "name": "Signing Key",
+          "description": "The secret key used to sign webhooks, provided by the Convex LaunchDarkly component."
```
Comment (atrakh) [inline integrations/convex/manifest.json:31]: Can the feature store sign payloads, similar to LaunchDarkly's audit log webhooks?

File: integrations/convex/manifest.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        "parser": {
+          "success": "/success",
+          "errors": "/errors"
+        }
+      },
+      "featureStoreRequest": {
+        "endpoint": {
+          "url": "{{deploymentPayloadUrl}}",
+          "method": "PUT"
+        }
```
Comment (atrakh) [inline integrations/convex/manifest.json:48]: These validationRequest and featureStoreRequest should also include the webhook signature passed as header, but the name and format of the header is undecided

File: integrations/convex/manifest.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  },
+  "categories": ["infrastructure"],
+  "capabilities": {
+    "featureStore": {
+      "formVariables": [
+        {
+          "key": "deploymentPayloadUrl",
+          "name": "Webhook URL",
+          "description": "The url to the webhook URL exposed by the LaunchDarkly component for Convex.",
+          "type": "string",
```
Comment (ldhenry) [inline integrations/convex/manifest.json:20]:
```suggestion
          "type": "uri",
```

**Review by ldhenry — CHANGES_REQUESTED** [review]
This looks good to me overall. I just have one small suggestion before we merge and enable for your account.

File: integrations/convex/manifest.json [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  "version": "0.0.1",
+  "overview": "Sync your LaunchDarkly feature flags to your Convex deployment.",
+  "description": "Convex is the open-source backend for application builders. An all-in-one platform with thoughtful, product-centric APIs.",
+  "author": "Convex",
+  "supportEmail": "support@convex.dev",
+  "links": {
+    "site": "https://convex.dev",
+    "privacyPolicy": "https://convex.dev/privacy"
+  },
+  "categories": ["infrastructure"],
```
Comment (ldhenry) [inline integrations/convex/manifest.json:12]:
```suggestion
  "categories": ["infrastructure"],
  "icons": {
      "square": "assets/images/square.svg",
      "horizontal": "assets/images/horizontal.svg"
    },
```

You'll need to specify the paths to the icons. To validate the manifest you you can run `npm run test`.

**Review by ldhenry — CHANGES_REQUESTED** [review]

File: integrations/convex/manifest.json (hunk 4 above)
Comment (atrakh) [reply integrations/convex/manifest.json:12]: whoops! I thought i had those specified

File: integrations/convex/manifest.json [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  "icons": {
+      "square": "assets/images/square.svg",
+      "horizontal": "assets/images/horizontal.svg"
+    },
```
Comment (ldhenry) [inline integrations/convex/manifest.json:16]: I think prettier is getting tripped up on the spacing here. Can you do me a favor and run `npm run prettier:write`

**Review by ldhenry — CHANGES_REQUESTED** [review]
Just one more formatting suggestion.

**Review by ldhenry — APPROVED** [review]

**Review by InTheCloudDan — APPROVED** [review]

**Pull request merged**